src/SignalR/clients/ts/signalr/src/HubConnection.ts

```diff
@@ -120,6 +120,17 @@ export class HubConnection {
         return this.connectionState;
     }
 
+    /** Represents the connection id of the {@link HubConnection} on the server. The connection id will be null when the connection is either
+     *  in the disconnected state or if the negotiation step was skipped.
+     */
+    get connectionId(): string | null {
+        if (this.connection.connectionId) {
```

Comment:
Does `return this.connection.connectionId || null` work here? (I'm guessing the if is to go from string/undefined to string/null)